src/main/java/net/minecraftforge/common/extensions/IForgeBlock.java

```diff
@@ -951,4 +952,15 @@ default BlockState getAppearance(BlockState state, BlockAndTintGetter level, Blo
     {
         return state;
     }
+
+    /**
+     * @deprecated call the method on BlockState
```

Comment:
This is not a Javadoc. Please write something proper explaining what this method is and what the return values mean.